puppet/zulip/manifests/common.pp

```diff
@@ -60,18 +60,18 @@
 
     # https://go.dev/dl/
     'golang' => {
-      'version' => '1.18.2',
+      'version' => '1.18.4',
       'sha256' => {
-        'amd64'   => 'e54bec97a1a5d230fc2f9ad0880fcbabb5888f30ed9666eca4a91c5a32e86cbc',
-        'aarch64' => 'fc4ad28d0501eaa9c9d6190de3888c9d44d8b5fb02183ce4ae93713f67b8a35b',
+        'amd64'   => 'c9b099b68d93f5c5c8a8844a89f8db07eaa58270e3a1e01804f17f4cf8df02f5',
+        'aarch64' => '35014d92b50d97da41dade965df7ebeb9a715da600206aa59ce1b2d05527421f',
       },
     },
 
     # https://github.com/stripe/smokescreen/tags
     'smokescreen-src' => {
-      'version' => 'e196cd61a007e1cdbde5579fe2dac95e7154190c',
+      'version' => '40870920671b64102c45b164152aef83a08d4735',
```

Comment:
`e196cd6` was the `v0.0.4` tag. Do we want to move to an unreleased version?

Reply:
They mostly only do numbered releases when prodded.  There are only three commits since 0.0.4, and one of them is a no-op from me.  The other two look reasonable.